openslides/core/models.py

```diff
@@ -152,6 +163,25 @@ def need_full_update_for(self, collection_element):
         return True
 
 
+class ProjectionDefault(RESTModelMixin, models.Model):
+    """
+    Model for the ProjectionDefaults like Motion, Agenda, List of speakers,...
+    """
+    access_permissions = ProjectionDefaultAccessPermissions()
+
+    name = models.CharField(max_length=256)
```

Comment:
Please add some comments or better extend the docstring about what name and display name should be here. Which choices are possible?